tests/analysis.spec.ts

```diff
@@ -61,6 +62,9 @@ describe('Functional test of analysis', () => {
             paths: [sampleProjectPath],
             symlinksEnabled: false,
           },
+          reportOptions: {
```

Comment:
q: See comment above about this being optional - we should require **no changes to existing tests**.